ExcelMakerConsole/ExcelMakerConsole.csproj

```diff
@@ -55,15 +55,24 @@
     <ManifestKeyFile>ExcelMakerConsole_TemporaryKey.pfx</ManifestKeyFile>
   </PropertyGroup>
   <PropertyGroup>
-    <GenerateManifests>true</GenerateManifests>
+    <GenerateManifests>false</GenerateManifests>
   </PropertyGroup>
   <PropertyGroup>
     <SignManifests>true</SignManifests>
   </PropertyGroup>
+  <PropertyGroup>
+    <TargetZone>LocalIntranet</TargetZone>
+  </PropertyGroup>
+  <PropertyGroup>
+    <ApplicationManifest>Properties\app.manifest</ApplicationManifest>
+  </PropertyGroup>

```

Comment:
I think these changes are giving me this error

![image](https://user-images.githubusercontent.com/7853605/54650101-3b71dd80-4a83-11e9-9b2a-fc405231bb42.png)

pry just junk from the all-powerful IDE